consensus/src/dag/dag_handler.rs

```diff
@@ -160,6 +160,8 @@ impl NetworkHandler {
                             Ok(certified_node) => {
                                 if let Err(e) = dag_driver_clone.process(certified_node).await {
                                     warn!(error = ?e, "error processing certified node fetch notification");
+                                } else {
+                                    dag_driver_clone.try_order_all();
```

Comment:
we may want to call it even it's an error?

Reply:
discussed offline. If there is an error anywhere in the call, the node doesn't get into the dag store so don't need to be ordered.